### DISCUSSION TITLE
Svelte Contribution

### DISCUSSION THREAD

**Discussion by seanaye** [tailwindlabs/headlessui#403] [General]
Great work on this project. Would you accept a PR for Svelte support?

**Comment by techgems** [comment]
I'm willing to help out too. Would love to see official svelte support on this.

**Reply by AndrewEQ** [reply]
Ditto!

**Reply by fabianmossberg** [reply]
Me too

**Comment by grandslammer** [comment]
+1

**Comment by ghost** [comment]
25800

در تاریخ چهارشنبه ۱۶ ژوئن ۲۰۲۱،‏ ۱۸:۲۷ Ivan ***@***.***> نوشت:

**Comment by rgossiaux** [comment]
For anyone following this thread who may be interested, I've just released an unofficial full Svelte port of this library: https://github.com/rgossiaux/svelte-headlessui

Same components with the same API and same implementation. Hope this helps!

**Comment by sebastianconcept** [comment]
+1. We need this

**Comment by vorant94** [comment]
it seems that the response that discussions with requests of support for svelte / web-components are getting (e.g. "currently we don't have the capacity to do it") actually mean that the core team has no capacity neither to implement it by themselves nor to review and accept a community contribution...

I doubt unofficial forks would get the same adoptions as the official support could have.

it happens that open source nowadays means a lot of different things: from a fully-community driven projects to the ones that are open source just for the sake of being open source...

**Reply by codeforfunzz** [reply]
+1 We need this, Svelte is better
